README.md

```diff
@@ -253,9 +255,17 @@ json.cache_if! !admin?, ['v1', @person], expires_in: 10.minutes do
 end
 ```
 
-If you are rendering fragments for a collection of objects, have a look at
-`jbuilder_cache_multi` gem. It uses fetch_multi (>= Rails 4.1) to fetch
```

Comment:
I do not believe we have to mention it any more as Jbuilder will have it built-in once this PR is merged. So I just replaced it with how `cached: true` could be used on collection rendering.